src/api/gasStation.ts

```diff
@@ -0,0 +1,65 @@
+import { walletApi } from './'
```

Comment:
It looks problematic this dependency. If you depend in `WalletApi`, the gas station should be a "factory function" with `WalletApi` as one of it's params.

Reply:
Done